controllers/admin/AdminAttributesGroupsController.php

```diff
@@ -615,7 +615,13 @@ public function initToolbarTitle()
                             $bread_extended[] = '<a href="'.Context::getContext()->link->getAdminLink('AdminAttributesGroups').'&id_attribute_group='.$id.'&viewattribute_group">'.$obj->name[$this->context->employee->id_lang].'</a>';
                         }
                         if (Validate::isLoadedObject($obj = new Attribute((int)$this->id_attribute))) {
-                            $bread_extended[] =  sprintf($this->trans('Edit: %s', array('%s' => $obj->name[$this->context->employee->id_lang]), 'Admin.Catalog.Feature'));
+                            $bread_extended[] =  $this->trans(
+                                'Edit: %value%',
+                                array(
+                                    '%svalue%' => $obj->name[$this->context->employee->id_lang]
```

Comment:
The place holder is incorrect, you forgot to remove a `s`.

`'%svalue%'` --> `'%value%'`